great_tables/_utils_render_latex.py

```diff
@@ -0,0 +1,1001 @@
+from __future__ import annotations
```

Comment:
It seems like there are some larger untested blocks. Can you take a look at this file in codecov, and test parts of some of the bigger blocks?

If rendering row / group names is unsupported, can you ensure it raises an error? (that path appears untested)

https://app.codecov.io/gh/posit-dev/great-tables/pull/481/blob/great_tables/_utils_render_latex.py

Reply:
Now, pretty much everything added in this PR has tests.

Reply:
Errors are raised for row / group names usage. And that's formally covered in unit tests as well.